dev/diagrams/on-dev-token-sign-in-handlers.md

```diff
@@ -1,27 +1,27 @@
-# On Dev Token Sign In Handlers
+# On Dev Sign In
 
 <!-- TOC -->
 
-- [On Dev Token Sign In Handlers](#on-dev-token-sign-in-handlers)
+- [On Dev Sign In](#on-dev-sign-in)
   - [Overview](#overview)
 
 <!-- /TOC -->
 
 ## Overview
-This is a basic activity diagram for the useDevTokenSignInHandlers.
+This is a basic activity diagram for the useDevSignIn.
 
 
 ```plantuml
 
 @startuml
 
-title useDevTokenSignInHandlers()
+title useDevSignIn()
 
 participant "End User" as user
 
 box "FE" #Lightblue
   participant "Component" as fe
-  participant "Hook: useDevTokenSignInHandlers" as hook
+  participant "Hook: useDevSignIn" as hook
   participant "Hook: UseAuthPrep" as hook2
```

Comment:
I think you can match lowercase here too
```suggestion
  participant "Hook: useAuthPrep" as hook2
```

Reply:
handled in https://github.com/ajktown/docs/pull/57/commits/bea4bf158977a509a87dcff934a242755a678337